### DISCUSSION TITLE
inference of detectron2 on CPU

### DISCUSSION THREAD

**Discussion by SylvainArd** [facebookresearch/detectron2#3498] [Q&A]
Hello, I have trained my data on GPU with maskrcnn and ResNet 101 and want to deploy it on CPU by demo.py. The code is written for GPU, how can I change it to works on CPU please ? Thank you !
Best regards

**Comment by sukjunhwang** [comment]
Adding ```MODEL.DEVICE cpu``` at the end of your command line will change the device to cpu.
You can check this at
https://github.com/facebookresearch/detectron2/blob/2a1cec4c647dbe8a05f2d5b599d7d4c7cf9f04ad/detectron2/config/defaults.py#L32
